### DISCUSSION TITLE
XCH is gone

### DISCUSSION THREAD

**Discussion by Jacktsai6888** [Chia-Network/chia-blockchain#5029] [Support]
![image](https://user-images.githubusercontent.com/84209645/118339718-441faa80-b54c-11eb-913e-896cafe24994.png)

5/14 11:07:59 earn 1.75/0.25 xch

The wallet is currently syncing

After synchronization is complete

xch is gone

how can i trace xch

it is my wallet address

xch1u50uqat89k5455yf34rfzxsp8nguw7sd5et9pjn9jtm90sznedcq4spadh

**Comment by cccat6** [comment]
Check it here www.chiaexplorer.com
By the way, if your wallet did not get synced, it might show 0.

**Reply by Jacktsai6888** [reply]
thanks

i have already check www.chiaexplorer.com
but still xch have nothing
by the way, my wallet is already synced ready
and the same thing happen with my other friend

**Comment by lvcivs** [comment]
chiastatus show a balance of zero for your wallet: https://www.chiastatus.com/addresses/xch1u50uqat89k5455yf34rfzxsp8nguw7sd5et9pjn9jtm90sznedcq4spadh

It does not show the farming reward from 5/14 that you apparently saw in the GUI. Not sure what's going on here.

**Reply by Jacktsai6888** [reply]
thanks for your reply

it is very stranger

![image](https://user-images.githubusercontent.com/84209645/118363215-e760d600-b5c5-11eb-9213-150689a0848a.png)

/he history shows i got  1.75/0.25 xch, but it is not showing in my wallet

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
